### DISCUSSION TITLE
WG_DEFAULT_ADDRESS

### DISCUSSION THREAD

**Discussion by vpsceo** [wg-easy/wg-easy#1070] [Q&A]
How can we be greater than 254 addresses here

**Comment by mitrios-hub** [comment]
No way, WG-EASY supports only CIDR/24 -> https://github.com/wg-easy/wg-easy/issues/430
the WG-EASY scripts doesn't support CIDR/16 but wireguard itself does - you can configure WG manually without using wg-easy

**Comment by kaaax0815** [comment]
Fixed with #1347
